apps/api/src/main.ts

```diff
@@ -42,11 +44,32 @@ class CustomLogger implements LoggerService {
   }
 }
 
-async function bootstrap() {
+async function initializeSentry() {
+  if (!process.env.SENTRY_DSN) {
+    Logger.error('Missing environment variable: SENTRY_DSN')
+    process.exit(1)
+  }
+
+  Sentry.init({
+    dsn: process.env.SENTRY_DSN,
+    enabled: process.env.NODE_ENV !== 'test' && process.env.NODE_ENV !== 'e2e',
+    environment: process.env.NODE_ENV || 'production',
```

Comment:
should have been `process.env.SENTRY_ENV || 'production'.` I want to explicitely declare this value

Reply:
Noted. Will update it as soon as time permits.